app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy != 0:
```

Comment:
You don't need to compare `masks_to_buy` to `0` directly. You can simplify your code by just checking `masks_to_buy` in the if statement. Python considers zero as False and any non-zero value as True in a boolean context. So, you could write `if masks_to_buy:` instead of `if masks_to_buy != 0:`. This is a code efficiency issue.